step3/index.html

```diff
@@ -83,6 +83,20 @@
   set image(image) {
     const canvas = this.root.getElementById('canvas');
 
+	//resize image to something reasonable
+	const maxsize = 1024;
+	if (image.height > image.width & image.height > maxsize){
```

Comment:
`&` probably works (unintentionally—it's bitwise AND) but I think you want `&&`

Reply:
Oh right. It's been a while since I've Javascripted :\